base/database/query.go

```diff
@@ -15,8 +15,9 @@ type AttrParser func(string) (interface{}, error)
 
 type AttrName = string
 type AttrInfo struct {
-	Query  string
-	Parser AttrParser
+	DataQuery  string
```

Comment:
For historic reasons I would keep just `Query` here. So we don't need to update all "query" annotations.

Reply:
It's npt changed in the annotations, just clarifies it in the reflection code.